ts/utils/messages.ts

```diff
@@ -243,7 +244,7 @@ const extractCTA = (
   serviceMetadata: MaybePotMetadata
 ): Option<CTAS> =>
   fromPredicate((t: string) => FM.test(t))(text)
-    .map(m => FM<MessageCTA>(m).attributes)
+    .chain(m => tryCatch(() => FM<MessageCTA>(m).attributes))
```

Comment:
The fix lgtm, do you think it could be useful to trace the case in which malformed messages arrive?

Reply:
I think so!
But, honestly, I don't know what we could trace, since the message is about personal data.

Do you think about a generic "frontmatter decoding exception"?